set.mm

```diff
@@ -58687,6 +58687,130 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
     ( c0 cpred ccnv csn cima cin df-pred 0in eqtri ) CABDCAEBFGZHCCABILJK $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Founded partial induction
```

Comment:
```suggestion
  Well-founded induction
```

Reply:
I want to differentiate this from the section immediately below it. That's also called "Well founded induction" but it concerns the total ordering version

Reply:
I see. But the adjective is really "well-founded", not "founded", and the induction is not "partial" (the hypothesis is that we have a "partial" order, but it does not translate straightfowardly).

Maybe we can find names for both sections after merging this one ? (I think the present section is more fundamental than the one already in set.mm, and they may later be merged.)